contributions/api/controllers/configs.py

```diff
@@ -33,7 +33,4 @@
 CAPABILITY_COLL_NAME = os.getenv('CAPABILITY_COLL_NAME', 'capabilities')
```

Comment:
```suggestion
```
We can also remove this as it is not used. Thanks.

Reply:
removed